HIP/hip-0000-token-update-remove-keys.md

```diff
@@ -0,0 +1,173 @@
+---
+hip: 0000
+title: Remove Existing Keys From A Token
+author: Justyn Spooner <justyn@dovu.io>
+working-group: Justyn Spooner <justyn@dovu.io>
+type: Standards Track
+category: Application
+needs-council-approval: Yes
```

Comment:
```suggestion
needs-council-approval: No
```

Reply:
Why doesn't this need council approval? 

Otherwise who is deciding to add the feature, which is a change to the way the network works?